ports/stm32f4/boards/pybv1_1/mpconfigboard.mk

```diff
@@ -0,0 +1,19 @@
+USB_VID = 0x239A
+USB_PID = 0x8056
+USB_PRODUCT = "Pyboard Version 1.1"
+USB_MANUFACTURER = "STMicroelectronics"
```

Comment:
Shouldn't this be George Robotic? (Damien's company)

Reply:
Oh yes, I was going to ask about this. The VID and PID won't be correct either, and I forgot to check them before wiping the firmware. What should I put down for that?